read-fonts/src/tables/layout.rs

```diff
@@ -262,6 +279,11 @@ impl CoverageFormat1<'_> {
     pub fn population(&self) -> usize {
         self.glyph_count() as usize
     }
+
+    /// Return the cost of looking up a glyph in this table
+    pub fn cost(&self) -> u32 {
+        bit_storage(self.glyph_array().len())
```

Comment:
if we do `bit_storage(self.glyph_count() as usize)` then we avoid the cost of reading/validating the array here